compiler-plugin-tests/src/test/java/io/ballerina/stdlib/websub/codeaction/AbstractCodeActionTest.java

```diff
@@ -55,13 +53,13 @@
 public abstract class AbstractCodeActionTest {
     private static final Gson GSON = new Gson();
 
-    @Test(dataProvider = "testDataProvider")
-    public void testCodeActions(String srcFile, int line, int offset, String resultFile)
-            throws IOException {
-        Path srcPath = getResourcePath("ballerina_sources", getTestPackage(), srcFile);
-        Path targetPath = getResourcePath("codeaction", getConfigDir(), resultFile);
-        performTest(srcPath, LinePosition.from(line, offset), targetPath);
-    }
+//    @Test(dataProvider = "testDataProvider")
```

Comment:
Shall we create a issue for this?

Reply:
May be we can reenable and see. If fail , we can disable (without commenting ) and create an issue

Reply:
It seems it is still failing with an OOM error.

Reply:
Disabled the test and created https://github.com/ballerina-platform/ballerina-library/issues/7373